packages/core/src/GreenButton.js

```diff
@@ -1,7 +1,11 @@
-import styled from 'styled-components'
+import React from 'react'
 import Button from './Button'
 
-const GreenButton = styled(Button).attrs({ color: 'secondary' })``
+const GreenButton = ({ children, ...props }) => (
+  <Button color="secondary" {...props}>
+    {children}
+  </Button>
+)
```

Comment:
Should we add a deprecation warning, saying that this will be removed in v3?

Reply:
absolutely - just needed to fix it for now, since it broke with styled-components 4 which is an accepted peer